packages/patternfly-4/react-core/src/components/Nav/Nav.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Nav/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /** NavList elements */
+  children(props, propName, componentName) {
+    const prop = props[propName];
+
+    let error = null;
+    React.Children.forEach(prop, child => {
+      if (child.type.componentType !== 'NavList') {
+        error = new Error(`${componentName} children should be of type NavList.`);
+      }
+    });
+    return error;
+  },
+  /** Additional classes added to the Nav */
+  className: PropTypes.string,
+  /** Non-visible label to describe the nav */
+  ariaLabel: PropTypes.string
```

Comment:
I think we want to at least warn the consumer that an aria label is required here.  Maybe a function similar to TextInput.

Reply:
Also, we have been using aria-label for the prop name in other components

Reply:
I'll remove this prop and let the user pass in their own aria-label if they choose.
@jgiardino should aria-label be required for the nav? didn't see it mentioned in the core nav docs.